docs/reference/migration/migrate_2_0.asciidoc

```diff
@@ -125,6 +125,12 @@ The `histogram` and the `date_histogram` aggregation now support a simplified `o
 `post_offset` rounding options. Instead of having to specify two separate offset shifts of the underlying buckets, the `offset` option
 moves the bucket boundaries in positive or negative direction depending on its argument.
 
+The `date_histogram` options for `pre_zone` and `post_zone` are replaced by the `time_zone` option. So far the `time_zone` option was
```

Comment:
I think this paragraph can be simplified to something like:

```
The `date_histogram` options for `pre_zone` and `post_zone` have been replaced by the `time_zone` option.  The behavior of `time_zone` is equivalent to the former `pre_zone` option.
```

I think the migration guide should be to the point about what options were available before, and are available now.